index.html

```diff
@@ -232,10 +232,10 @@ <h3 class="mainText CreteRoundfont">About me</h3>
     </section>
 
     <footer class="footer">
-      <div class="Interfont footerTxt">
+      <p class="Interfont footerTxt">
```

Comment:
Nice job implementing the changes, though I noticed the text is no longer in the center of the page, you can apply `width: 100% `and `text-align: center` to your `p`  to fix

This is how your text current appears. kindly fix as suggested

![image](https://user-images.githubusercontent.com/43564324/165742558-27f44955-ea3a-4417-988d-f0d9a1e0679a.png)